### DISCUSSION TITLE
Guidance on using collection vs. card group

### DISCUSSION THREAD

**Discussion by patrickcate** [uswds/uswds#4829] [Q&A]
Hello,

I'm working on a new USWDS site and its not clear to me where the collection component should be used versus a card group. There seems to be some overlap in the guidance for both. Adding some information for when to use one over the other would be great.

Thanks for your hard work on USWDS.

**Comment by mejiaj** [comment]
Hey Patrick! It really depends on what your content is. Digital.gov, for example uses both components in their [Communities landing page](https://digital.gov/communities/).

**Comment by ryparker** [comment]
A useful distinction is the amount of content each item needs:

- Use a [collection](https://designsystem.digital.gov/components/collection/) for a compact list of related links with optional summaries and metadata. For example, recent announcements with publication dates, or upcoming events.
- Use [cards](https://designsystem.digital.gov/components/card/) when each item needs a more self-contained presentation, such as a heading, image, description, and prominent action. For example, three featured services on a landing page.

There is overlap: either can link to articles or resources. Choose the structure that makes the content easiest to scan, rather than choosing solely because one has a border or a grid. Cards should be understandable independently; neither component replaces a table when users need to compare structured data across rows.
